src/components/HURUmap/Map/Layers.js

```diff
@@ -0,0 +1,141 @@
+import L from "leaflet";
+import { useRouter } from "next/router";
+import PropTypes from "prop-types";
+import React, { useEffect, useRef } from "react";
+import { useMap, LayerGroup, FeatureGroup, GeoJSON } from "react-leaflet";
+
+const geoStyles = {
+  inactive: {
+    fillColor: "#f8f8f8",
+    fillOpacity: 1,
+    color: "#666666",
+    weight: 1,
+  },
+  hoverOnly: {
+    over: {
+      fillColor: "#7DB2D3",
+      fillOpacity: 1,
+      color: "#666666",
+    },
+    out: {
+      fillColor: "#DFDFDF",
+      fillOpacity: 1,
+      color: "#666666",
+      weight: 1,
+    },
+  },
+  selected: {
+    over: {
+      color: "#666666",
+      fillColor: "#7DB2D3",
+      opacity: 1,
+    },
+    out: {
+      color: "#666666",
+      fillColor: "#7DB2D3",
+      strokeWidth: 1,
+      opacity: 1,
+      fillOpacity: 1,
+
+      weight: 1.5,
+    },
+  },
+};
+
+const Layers = ({
+  selectedBoundary,
+  parentsGeometries,
+  setGeoCode,
+  setShouldFetch,
+}) => {
+  const map = useMap();
+  const router = useRouter();
+  const groupRef = useRef();
+
+  const featuredCountiesCode =
+    process.env.NEXT_PUBLIC_FEATURED_COUNTIES?.split(",");
+
+  const popUpContent = (level, name) =>
+    `<div class='tooltip'><div class='level'>${level}</div> <div class='name'>${name.toLowerCase()}</div></div>`;
```

Comment:
Can't this be a React component that we [render as String](https://reactjs.org/docs/react-dom-server.html#rendertostaticmarkup) or something? My thinking being this component is similar/exactly the same used on the map heads-up display of location data.
![Screenshot from 2021-09-10 13-34-33](https://user-images.githubusercontent.com/1779590/132840946-2f15b254-9634-40d0-bfcf-722080e2635b.png)